parameter-names/src/test/java/com/fasterxml/jackson/module/paramnames/ParameterNamesAnnotationIntrospectorTest.java

```diff
@@ -9,16 +9,16 @@
 import org.junit.runner.RunWith;
 import org.mockito.BDDMockito;
 import org.mockito.Mock;
-import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.junit.MockitoJUnitRunner;
```

Comment:
is this needed on any of the other tests with the mockito version bump?

Reply:
this appears to be the only test in this code set that uses mockito

Reply:
Would be great to remove the dependency if it's the only usage. But not sure how much work that would be.

Reply:
the test seems to fail with that this

Reply:
the older (original) mockito jar did not support latest java versions